dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h

```diff
@@ -25,6 +17,26 @@ void LogErrorInMemory(HRESULT hr, ULONG line, _In_ PCSTR filename, _In_ PCWSTR m
 #define LOG_ERROR_IN_MEMORY(hr, line, filename, message) __noop
 #endif
 
+class MrtRuntimeTraceLoggingProvider : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(MrtRuntimeTraceLoggingProvider, "Microsoft.Windows.Mrt.Runtime",
```

Comment:
>"Microsoft.Windows.Mrt.Runtime" [](http://example.com/codeflow?start=65&length=31)

rename

Reply:
@ZenBird thought we should keep the same name as the Windows provider. I'm OK with both.

Reply:
Yes. This should be kept the same between Windows SDK and WindowsAppSDK. Easier for reporting.

Reply:
Would this end up producing confusing data? Like, if we collect the same data point in both, how do we distinguish?

Reply:
The data logged using this framework will have PartB that will not be present if the data is logged by WindowsSDK.

Reply:
I agree with Axel, it will be confusing. How is this going to be easier for reporting? What is the use case? This will make streams need an extra field to purposefully put down the fact that it came from the SDK. Which is why the provider can be named more explicitly and events named differently unless they signify the exact same thing. 
Also, if there is Part C data (which majority of events carry), now you need to look at B to know what you expect from C.

Reply:
Joel, any preferred name for the provider? Seems to me that it would be good for the components to use a consistent naming scheme. How's `Microsoft.WindowsAppSdk.MrtCore.Runtime`?

Reply:
Isn't this how Closed-source Reunion components are logging telemetry. I do not believe they are changing names. @jvarela64 : you are closer to that side of things. I will let you make the decision.

Reply:
The existing closed source Reunion telemetry will have to live with those names or data pipelines will break. Unless we make a concerted and risky change to both code and pipelines (I just updated event parsing for Windowing which luckily didn't involve much). 
Since we have the opportunity here to set the precedent for the new App SDK telemetry, it should be named appropriately. The name which Rohan suggests sounds correct to me.

Reply:
Done!